lib/skate/detours/db/detour.ex

```diff
@@ -7,6 +7,7 @@ defmodule Skate.Detours.Db.Detour do
   alias Skate.Settings.Db.User
 
   schema "detours" do
+    field :uuid, :string
```

Comment:
question: would it be possible to use the table `id` instead of creating a new column? I think we should be able to configure the ID to be a UUID, and I think we should have the database create the ID and not the frontend.

Reply:
Oh nvm! I see your subsequent comment explains this exact process. Yeah, makes sense to me

Reply:
that sounds correct.